ublox-bluetooth-commands.go

```diff
@@ -164,6 +164,11 @@ func (ub *UbloxBluetooth) ConnectToDevice(address string, onConnect DeviceEvent,
 	var d []byte
 	var cr *ConnectionReply
 
+	err = ub.ConfigureSecurity(oobTemporaryKey, securityEnabledOutOfBand)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Martin has asked me to make this configurable so we are not finished yet on this review

Reply:
Sensibly he said the same as you, Encryption is a pain and things will go wrong before they go right..